scotty

```diff
@@ -105,7 +105,13 @@ def create_venv():
 
     if not os.path.isdir(venv_real_dir):
         print_color("\nCreating virtual env in %s..." % Config.get("venv"))
-        run_commands(["%s %s" % (str(Config.get("virtualenv")), str(venv_real_dir))])
+
+        # if scotty cannot find the virtual environment command at the specified path
+        # then just call the function in a hope that it is included in the path.
+        try:
+            run_commands(["%s %s" % (str(Config.get("virtualenv")), str(venv_real_dir))])
+        except:
+            run_commands(["%s %s" % (str("virtualenv"), str(venv_real_dir))])
```

Comment:
hmm again put in another PR? Then just have Ben take 5 seconds to look at it and merge it